set.mm

```diff
@@ -254514,12 +254510,23 @@ This function maps an element of the formal polynomial algebra (with
       TWGJVDUDXAVIWHABHWFVEAWGIVJWSAIIFIBHSSAIUHBWQVFAICHQVFRRWRVGIWGVKUMWTAJEV
       LNVMVTABVCAWPWOBVCUFUGPRDGISBKVNUSVOAUBBHICSXAFUHSVCJAXAVPWMVCWLFUDJVQACF
       EWLJLNMVRVSWQQRVCSTAWAVTWBXAWGSSJWCWDWE $.
+  $}
 
+  ${
+    $d B h $.  $d I h $.
+    psrbagev2.d $e |- D = { h e. ( NN0 ^m I ) | ( `' h " NN ) e. Fin } $.
+    psrbagev2.c $e |- C = ( Base ` T ) $.
+    psrbagev2.x $e |- .x. = ( .g ` T ) $.
+    psrbagev2.t $e |- ( ph -> T e. CMnd ) $.
+    psrbagev2.b $e |- ( ph -> B e. D ) $.
+    psrbagev2.g $e |- ( ph -> G : I --> C ) $.
+    psrbagev2.i $e |- ( ph -> I e. _V ) $.
     $( Closure of a sum using a bag of multipliers.  (Contributed by Stefan
        O'Rear, 9-Mar-2015.)  (Proof shortened by AV, 18-Jul-2019.) $)
     psrbagev2 $p |- ( ph -> ( T gsum ( B oF .x. G ) ) e. C ) $=
-      ( cof co cvv wf cfsupp wbr psrbagev1 simpld simprd gsumcl ) AICBHFSTZEUAJ
-      LNORAICUIUBZUIJUCUDZABCDEFGHIJKLMNOPQRUEZUFAUJUKULUGUH $.
+      ( cof co cvv c0g cfv eqid wf cfsupp wbr psrbagev1 simpld simprd gsumcl )
+      AICBHFQRZESETUAZKUKUBZMPAICUJUCZUJUKUDUEZABCDEFGHIUKJKLULMNOPUFZUGAUMUNUO
+      UHUI $.
```

Comment:
Why is this proof longer than the old one?

Reply:
the hypothesis [psrbagev1.z](https://us.metamath.org/mpeuni/psrbagev2.html) was replaced with an eqid